docs/bigquery-usage.rst

```diff
@@ -281,7 +281,7 @@ Run a query which can be expected to complete within bounded time:
    >>> from gcloud import bigquery
    >>> client = bigquery.Client()
    >>> query = """\
-   SELECT count(*) AS age_count FROM dataset_name.person_ages
+   SELECT count(*) AS age_count FROM [project-name:dataset_name.person_ages]
```

Comment:
**Confirmed**: After `time.sleep(90)` the results came back as expected. Maybe we should just nuke this PR?

In reality I was trying to find a way to get the "knowledge" about `[...]` in queries into our code, but maybe this wasn't the right place?